metricbeat/module/prometheus/collector/_meta/data.json

```diff
@@ -1,25 +1,30 @@
 {
     "@timestamp": "2017-10-12T08:05:34.853Z",
-    "beat": {
+    "agent": {
         "hostname": "host.example.com",
         "name": "host.example.com"
     },
+    "event": {
+        "dataset": "prometheus.collector",
+        "duration": 115000,
+        "module": "prometheus"
+    },
     "metricset": {
-        "host": "prometheus:9090",
-        "module": "prometheus",
-        "name": "collector",
-        "namespace": "collector",
-        "rtt": 115
+        "name": "collector"
     },
     "prometheus": {
-        "collector": {
-            "label": {
-                "event": "add",
-                "role": "node"
-            },
-            "prometheus_sd_kubernetes_events_total": {
-                "value": 0
-            }
+        "http_request_duration_microseconds_count": 0,
+        "http_request_duration_microseconds_sum": 0,
+        "http_request_size_bytes_count": 0,
+        "http_request_size_bytes_sum": 0,
+        "http_response_size_bytes_count": 0,
+        "http_response_size_bytes_sum": 0,
+        "labels": {
+            "handler": "drop_series"
         }
+    },
+    "service": {
+        "address": "172.29.0.2:9090",
+        "type": "prometheus"
```

Comment:
This is an interesting one. We should probably make it configurable so the user can set service.type and service.name.